dts/bindings/led/nxp,pca9956.yaml

```diff
@@ -6,3 +6,12 @@ description: NXP PCA9956 LED
 compatible: "nxp,pca9956"
 
 include: i2c-device.yaml
+
+properties:
+    default-iref:
+        type: int
+        default: 255
```

Comment:
Not sure, but doesn't this default value of `255` always override `0` used in driver: `.default_iref = DT_PROP_OR(node_id, default_iref, 0),`?

Reply:
You are right, as property has default value, I removed `DT_PROP_OR`, it is COND_CODE anyway, so I assume it if safe enough.

Reply:
Ok, i remember now, i was thinking about exotic scenario where you have multiple different PCA chips in design, so `.default_iref` of config struct will be compiled and filled by macro, but non always will be present in dts, as it is part only of pca9956 chip, not the rest of them, in that case i wanted to `zero` this field.

Reply:
You may consider removing the default if that is appropriate.

If you need to keep it, you must justify why the choice of the default value was made **in the description in the binding** (i.e. GitHub comments with a justification **do not count**).

Otherwise bindings LGTM, thank you for your patience.

Reply:
I have removed this default value, and remade driver so skipping property in dts will just ignore it. Thanks.